modes/elixir-mode/lsp-config.lisp

```diff
@@ -0,0 +1,14 @@
+(defpackage :lem-elixir-mode/lsp-config
+  (:use :cl
+         :lem-lsp-mode
+         :lem-lsp-base/type))
+
+(in-package :lem-elixir-mode/lsp-config)
+
+(define-language-spec (elixir-spec lem-elixir-mode:elixir-mode)
+  :language-id "elixir"
+  :root-uri-patterns '("mix.exs")
+  :command '("sh" "language_server.sh")
```

Comment:
Is this program being put in place where the path goes?

Reply:
It's not usually in the path tho, but it also doesn't have like a default location, so I just put the string in a variable that the user can edit to make it work (this probably should me documented in some place)

Reply:
Done :+1: